repos.d/rpm/fedora.yaml

```diff
@@ -386,6 +386,47 @@
       url: 'https://src.fedoraproject.org/rpms/{srcname}/raw/f34/f/{srcname}.spec'
   tags: [ all, production, fedora, rpm ]
 
+- name: fedora_35
+  type: repository
+  desc: Fedora 35
+  statsgroup: Fedora
+  family: fedora
+  ruleset: [fedora, rpm]
+  color: '294172'
+  minpackages: 25000
```

Comment:
This fails as it only has 22581 packages. You should probably leave 22000 everywhere, is it's the fair number of packages without duplicates.

Reply:
Oh, this is about source packages… Yeah, I think we have 23.5k now.